packages/nextjs/src/config/types.ts

```diff
@@ -467,6 +467,13 @@ export type SentryBuildOptions = {
    */
   suppressOnRouterTransitionStartWarning?: boolean;
 
+  /**
+   * Disables manifest injection.
```

Comment:
l: I would prefer if we added some more details to this docstring about what a manifest is.